### ISSUE TITLE
Global identify events will cause packet loss

### ISSUE THREAD

**Issue by balazsracz** [bakerstu/openmrn#6]
The current implementation of global identify events is as follows:

- the event packet is picked up by the CANbus RX thread.
- it ends up in nmranet_if_rx_data
- which calls nmranet_event_packet_global
- that iterates over all virtual nodes and calls nmranet_event_packet_addressed on it
- each node in turn will traverse its event list
- and generate an identify message for each of them
- which will be put into the tx queue
- the TX queue will not be processed due to the likely packet collisions on the bus
- the TX queue will get full and the next write will block the RX thread
- while there are lots of incoming packets coming from the network from other nodes
- eventually overflowing the RX queue.

The first of those incoming packets will be processed only once all the event identify packets have been enqueued into the TX queue.

**Comment by bakerstu** [comment]
The fundamental architecture has changed. This issue is no longer relevant.
